pkg/public/strategies.go

```diff
@@ -0,0 +1,209 @@
+package public
```

Comment:
Can we add to the top of this file the why of these functions?

- We provide a utility to abstract the various strategy map ConfigMaps from the service using CRO (RHMI)
- We accept start times because different cloud providers may have different minimum lengths for backup and maintenance